rhombus/tests/check.rhm

```diff
@@ -0,0 +1,35 @@
+#lang rhombus
+
+// I used the check to check the check…
+check:
+  check: values(1, [2, 3])
+         ~matches values(a, [b, c])
+  ~completes
+
+macro 'with_err_to_string:
+         $body ...':
+  'block:
+     def old_port = Port.current_error()
+     def new_port = Port.open_output_string()
+     Port.current_error(new_port)
+     block:
+       $body ...
+     Port.current_error(old_port)
+     Port.get_output_string(new_port)'
+
+check:
+  def the_output = with_err_to_string:
+    check: values(1, [2, 3])
+           ~matches a
```

Comment:
A nit, but I encourage `def the_output:` over `def the_output =` when the part after `=` is multiple lines.